src/node_buffer.cc

```diff
@@ -92,6 +118,11 @@ size_t Length(Handle<Object> obj) {
   return obj->GetIndexedPropertiesExternalArrayDataLength();
 }
 
+Local<Object> WrapNullPointer(Environment* env) {
+  size_t buf_size = 0;
+  char *ptr = reinterpret_cast<char *>(NULL);
+  return Buffer::Use(env, ptr, buf_size);
+}
```

Comment:
Can be simplified to just `return Buffer::Use(env, nullptr, 0)`.  I'd go further and just do it inline at the only call site.